machine/state.go

```diff
@@ -25,7 +25,7 @@ const (
 type MachineState struct {
 	ID       string
 	PublicIP string
-	Metadata map[string]string
+	Metadata map[string]string `json:"-"`
```

Comment:
Is it due to how you're unmarshaling both the machine and the metadata into the same object independently in the registry?